exercises/list_links.md

```diff
@@ -0,0 +1,39 @@
+## Listsof links
+ 
+
+In this exercise, we're going to consolidate our knowledge from previous sections.
+
+
+Start with an empty unordered html list in the static html:
+
+```html
+<ul id="list">
+</ul>
+```
+
+Now we can **dynamically** add your favourite links to the html list. 
+> 🔑 **Dynamically** means **capable of action or change**, when used in programming.
```

Comment:
That's not what "dynamically" literally means here. Maybe something like "Dynamically", means using blocks, so that the html **changes** when you press run, as opposed to statically by just changing the html directly?